connector/rest/src/main/java/it/pagopa/selfcare/dashboard/connector/rest/model/user_registry/UserResponse.java

```diff
@@ -0,0 +1,18 @@
+package it.pagopa.selfcare.dashboard.connector.rest.model.user_registry;
+
+import it.pagopa.selfcare.dashboard.connector.model.user.Certification;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
@KevinSi96 use @Data instead of @Getter and @Setter so Lombok will create the toString() method too